assets/script.js

```diff
@@ -1,32 +1,41 @@
+let counter = 0;
 let bookList = [];
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
```

Comment:
- [ ] Great job working on the functionality so far but I think it would be great if you add all the methods inside the class to be class methods as mentioned in the requirements since the refactoring to classes include the methods to be inside the class but I could see that you just made a class with a constructor inside it so kindly add all the method inside it 🚀🚀🤩💯💪🏻

![image](https://github.com/chudisoft/awesome-books/assets/58553711/0cf305b7-c839-4cf6-aca2-82a911405c91)